docs/topics/modules/planning_development_work.adoc

```diff
@@ -1,54 +1,29 @@
 [id="planning_development_work"]
 = Planning your development work
 
-When you create a space, the *Create an Application* wizard is displayed. Close the wizard, so that you can plan an end to end development workflow for launching and deploying a sample application before you launch the application.
+When you create a space, the *Create an Application* wizard is displayed. Close the wizard, so that you can plan an end to end development workflow before you launch the application.
 
-{osio} planner provides a set of predefined templates based on common development methodologies such as Agile and Scenario Driven Development. The development process you select while creating the space determines the planner's guided work item type hierarchy.
+{osio} planner provides a set of predefined templates based on common development methodologies such as Agile and Scenario Driven Development. The development process you select while creating the space determines the guided work item type hierarchy in the planner.
 
-This example walks you through the Scenario Driven Development planning process to plan your work.
+This example walks you through the Agile planning process to plan your work.
 
-. In the dashboard of your space, select the *Plan* tab at the top of the page to view the default *Backlog* view of the planner.
-. Select the *Show Tree* check box to see the work items and their children in a tree view. The tree view allows you to quickly add work items according to the guided hierarchy using the quick-add and in-line add options.
-. Use the quick-add option to create a *Scenario* as follows:
-.. In the *Add Work Item* pane, select *Scenario* from the drop-down list.
-+
-image::quickadd_scenario.png[Quick add scenario]
-+
-.. In the *Work Item Title* field, add *Enable an end to end, developer work-flow, using the continuous integration and deployment framework* as the title.
-.. Press kbd:[Enter] to create the scenario.
+. In the dashboard of your space, select the *Plan* tab at the top of the page. The default *Backlog* view displays the *Work Items* for your space.
+. Select the *Show Tree* check box to use the quick-add and in-line add options to quickly create workitems and their children.
+. In the *Add Work Item* pane, select *Theme* from the drop-down list, type *Enable a complete developer work-flow in a continuous integration and delivery framework* as the title, and press kbd:[Enter] to create the theme.
 
-. Create a child *Experience* for the *Scenario* using the in-line add option as follows:
-.. In the work item list, click the (+) icon adjacent to the scenario to see the in-line *Add Child Work Item* pane below it.
+. Create a child *Epic* for the *Theme* as follows:
+.. In the work item list, click the (+) icon adjacent to the theme to see the in-line *Add Child Work Item* pane below it.
```

Comment:
Reiterating the above: here we (and the UI) calls it a child work item. Let's just stick to the terms work item and child work item to reduce confusion?

Reply:
I'm a bit confused now. Your linked document suggests that there are three work item types: Work Items (having a work item as a work item type is a bit odd in itself, but ignoring this for now), Tasks, and Iterations. I'm suggesting we refer to these (Work Items) as Work Items rather than as Themes, Epics, etc.

Reply:
@mishaone the linked doc merely gives definitions of the WITs in Agile, not necessarily in order. To map to SDD, Themes would map to Scenarios, Epics map to Experiences, and Story to Requirements. Tasks are children of stories.
As I explained earlier, calling them work items might confuse the user more, we are expecting the user to know codebases and pipelines. Themes, epics anduser stories are common agile terms used in most enterprises and we can expect most users to be familiar with them. In addition we would have infotips to explain them to the users.

Reply:
Yes, agreed, BUT these are just work items. We can call them Theme type Work Items, which is clearer because we are still talking about Work Items. I would suggest removing the problem and just simplifying the use case for tutorials because the idea is to introduce people to the basics, not overwhelm them with terminology. Bear in mind this is the first thing people are doing after creating a space so if this is too confusing, they may not want to continue. Simple and clear would be my recommendation here.

Reply:
Lets discuss this Misha, depending on the final intent and the task we want to highlight we can move this module somewhere down the guide. We dont break the flow too this way.